pkg/tools/sandbox/dial_options.go

```diff
@@ -57,7 +57,6 @@ func DialOptions(options ...DialOption) []grpc.DialOption {
 		),
 		grpc.WithBlock(),
 		grpc.WithDefaultCallOptions(
-			grpc.WaitForReady(true),
```

Comment:
@Bolodya1997 I removed this line because without it https://github.com/networkservicemesh/sdk/blob/07ced33cad484b6713a8d0b9497ed8a443f9cd9f/pkg/networkservice/chains/nsmgr/suite_test.go#L310-L356 fails.  In digging into it, it was failing because we had WaitForReady set to true... no idea why it didn't fail before.  Any insights?  Anything I missed here?

Reply:
What the error you see in Test_ConnectToDeadNSEUsecase with `grpc.WaitForReady(true)` ?

Reply:
I suppose that there are some reasons why we may want to use `WaitForReady` flag, but I am not actually sure because we most probably had started using it before I have started working on NSM :)
There was an issue with Request hanging in such case and so it is the reason why do we have such test.
We solved this issue in `connect` several times with several approaches during the `heal` development. Currently `connect` uses `heal` client (or some other [monitoring client](https://github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/common/connect/monitor_client_test.go)) to understand that remote side is dead and so close the gRPC connection to prevent Request hanging with `WaitForReady` flag set.

Reply:
@edwarnicke 
Any update for this?

Reply:
I've restored the grpc.WaitForReady(true)